src/Command/WorkingHourDailyCommand.php

```diff
@@ -43,13 +44,18 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         foreach ($p->elements as $elem) {
             $id=$elem['id'];
             $perso_id=$elem['perso_id'];
-            $db=new \db();
-            $db->CSRFToken = $CSRFToken;
-            $db->update('planning_hebdo', array('actuel'=>0), array('perso_id'=>$perso_id));
-            $db=new \db();
-            $db->CSRFToken = $CSRFToken;
-            $db->update('planning_hebdo', array('actuel'=>1), array('id'=>$id));
+            $planningHebdoActuel = $this->entityManager->getRepository(Workinghour::class)->findBy(['perso_id' => $perso_id]);
+            foreach($planningHebdoActuel AS $pla) {
+                $pla->setCurrent(0);
+                $this->entityManager->persist($pla);
+            }
+            $planningHebdo=$this->entityManager->getRepository(Workinghour::class)->find($id);
+            if($planningHebdo) {
+                $planningHebdo->setCurrent(1);
+                $this->entityManager->persist($planningHebdo);
+            }
```

Comment:
Create a new function named changeCurrent in WorkingHourRepository.
This function will use the query builder to make these two simple queries
```
UPDATE planning_hebdo SET current = 0 WHERE debut > SYSDATE() OR fin < SYSDATE();
UPDATE planning_hebdo SET current = 1 WHERE debut <= SYSDATE() AND fin >= SYSDATE();
```
Then call it witth `$this->entityManager->getRepository(Workinghour::class)->changeCurrent();`
This call will replace all the code from line 39 to 57 (keep `$this->entityManager->flush();`)